apps/client-ui/scripts/health-check-preview.sh

```
@@ -0,0 +1,314 @@
+    echo "  $0 123    # Check health of PR #123 preview environment"
+}
+
+# Function to check if URL is accessible
+check_url_health() {
+    local url="$1"
+    local description="$2"
+    
+    print_info "Checking $description: $url"
+    
+    # Check if URL responds with 200 status
+    local http_status=$(curl -s -o /dev/null -w "%{http_code}" --max-time $TIMEOUT "$url" 2>/dev/null || echo "000")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)